README.md

```diff
@@ -1,66 +1,55 @@
 # Cap'n Proto bindings for Go
 
-[![GoDoc](https://godoc.org/zombiezen.com/go/capnproto2?status.svg)][godoc]
-[![Build Status](https://travis-ci.org/capnproto/go-capnproto2.svg?branch=master)][travis]
+[![GoDoc](https://godoc.org/capnproto.org/go/capnp/v3?status.svg)][godoc]
+![License](https://img.shields.io/badge/license-MIT-brightgreen?style=flat-square)
```

Comment:
Tangentially related, it appears GitHub's auto-detection of licenses can't work out which license we use, probably due to formatting oddities.

Reply:
What would be the course of action, here?

Reply:
We could futz with it, but I don't care that much, just something I noticed when I saw this.